docs/managedosimage-reference.md

```diff
@@ -12,6 +12,8 @@ import ClusterTarget from "!!raw-loader!@site/examples/upgrade/upgrade-cluster-t
 # ManagedOSImage reference
 
 The ManagedOSImage resource is responsible of defining an OS image or image version that needs to be applied to each node in a set of targeted Clusters.  
+Once created, the ManagedOSImage resource can be updated with a new `osImage` or `managedOSVersionName` to trigger a following upgrade.  
```

Comment:
nit: the "following upgrade"  at first read was not crystal clear to me... what about changing to something like "new upgrade" or "a brand new upgrade workflow" ?
```suggestion
Once created, the ManagedOSImage resource can be updated with a new `osImage` or `managedOSVersionName` to trigger a new upgrade.  
```